### PR TITLE
prow: add shellcheck test for ironic-ipa-downloader

### PR DESCRIPTION
**Pull request by tuminoid**
Depends on https://github.com/metal3-io/ironic-ipa-downloader/pull/37 merged first, or it'll block all PRs for ironic-ipa-downloader.

### CODE REVIEW

**Comment by tuminoid** [comment]
https://github.com/metal3-io/ironic-ipa-downloader/pull/37 has been merged.

/assign furkatgofurov7

File: prow/config/config.yaml [hunk 1]
```diff
@@ -1023,3 +1023,18 @@ presubmits:
...
+        containers:
+          - args:
+              - ./hack/shellcheck.sh
+            command:
+              - sh
+            env:
+              - name: IS_CONTAINER
+                value: "TRUE"
+            image: docker.io/koalaman/shellcheck-alpine:stable
+            imagePullPolicy: Always
```
Comment (furkatgofurov7) [inline prow/config/config.yaml:1040]:
```suggestion

  metal3-io/ironic-ipa-downloader:
  - name: shellcheck
    run_if_changed: '((\.sh)|^Makefile)$'
    decorate: true
    spec:
      containers:
        - args:
            - ./hack/shellcheck.sh
          command:
            - sh
          env:
            - name: IS_CONTAINER
              value: "TRUE"
          image: docker.io/koalaman/shellcheck-alpine:stable
          imagePullPolicy: Always
```

**Review by furkatgofurov7 — COMMENTED** [review]
Nit: giving a blank space before repo name and a bit of indentation adjustment would be nice

File: prow/config/config.yaml (hunk 1 above)
Comment (tuminoid) [reply prow/config/config.yaml]: Good catch. My editor did auto-formatting that didn't match rest of the file. Fixed now.

**Review by furkatgofurov7 — APPROVED** [review]
Thanks @tuminoid!

/approve

**Review by kashifest — COMMENTED** [review]
/lgtm

**Pull request merged**